index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include pure JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed to the
+data model according to the rules provided in this section, they are considered
+<em>"compatible with the Verifiable Credentials ecosystem"</em>, and the result
+of the transformation, which is a <a>conforming document</a>, is a
+<a>verifiable credential</a>. Specifications that contain transformations to the
+data model described by this specification:
+        </p>
+
+        <ul>
+          <li>
+MUST identify if the transformation to this data model is uni-directional or
```

Comment:
again,  media types are more useful here... Because its obvious that some transformations are unidirectional... the most obvious example is:

`application/vc+ld+json` -> `application/n-quads`.

You have not commented on if this transformation is "lossy or not"... that seems very wise to do, and will also be obvious based on the media types... for example, you can express more in YAML and CBOR than you can in JSON, so any transformation from YAML or CBOR can be lossy.

I don't think its worth saying more here, just point out I agree with the ommision.

Reply:
```suggestion
MUST identify whether the transformation to this data model is one-way-only or
```

Reply:
@OR13 I have made it explicit which media types are allowed for a `conforming document` in bd9aab2a283bd45789984fe2fc528c696ebe2aac.